### ISSUE TITLE
MultiQC 

### ISSUE THREAD

**Issue by aledj2** [ewels/MultiQC#426]
Hi,
I am running MultiQC version 1.0.dev0 which I have installed as a conda environment. 
When I run this I don't get an output for fastqc and the below error message.

Using version 0.8.dev0 (installed from source not conda) the same files produce a fastqc report with no error message.

[NGS126D_40_136819_NA12878_FocusedExome_S29_R1_001_trimmed.stats-fastqc.txt](https://github.com/ewels/MultiQC/files/864292/NGS126D_40_136819_NA12878_FocusedExome_S29_R1_001_trimmed.stats-fastqc.txt)
[NGS126D_40_136819_NA12878_FocusedExome_S29_R2_001_trimmed.stats-fastqc.txt](https://github.com/ewels/MultiQC/files/864293/NGS126D_40_136819_NA12878_FocusedExome_S29_R2_001_trimmed.stats-fastqc.txt)

```
============================================================
Module fastqc raised an exception: Traceback (most recent call last):
  File "/home/aled/anaconda2/envs/multiqc_exp/lib/python2.7/site-packages/multiqc-1.0.dev0-py2.7.egg/EGG-INFO/scripts/multiqc", line 346, in multiqc
    output = mod()
  File "/home/aled/anaconda2/envs/multiqc_exp/lib/python2.7/site-packages/multiqc-1.0.dev0-py2.7.egg/multiqc/modules/fastqc/fastqc.py", line 110, in __init__
    self.gc_content_plot()
  File "/home/aled/anaconda2/envs/multiqc_exp/lib/python2.7/site-packages/multiqc-1.0.dev0-py2.7.egg/multiqc/modules/fastqc/fastqc.py", line 430, in gc_content_plot
    for f in self.find_log_files(config.sp['fastqc']['theoretical_gc']):
KeyError: 'theoretical_gc'
============================================================
```

Thanks!

**Comment by aledj2** [comment]
I have also run v1.0.dev0 on another data set and I got an additional error, this time in Picard:

```
Module picard raised an exception: Traceback (most recent call last):
  File "/home/aled/anaconda2/envs/multiqc_exp/lib/python2.7/site-packages/multiqc-1.0.dev0-py2.7.egg/EGG-INFO/scripts/multiqc", line 346, in multiqc
    output = mod()
  File "/home/aled/anaconda2/envs/multiqc_exp/lib/python2.7/site-packages/multiqc-1.0.dev0-py2.7.egg/multiqc/modules/picard/picard.py", line 76, in __init__
    n['AlignmentMetrics'] = AlignmentSummaryMetrics.parse_reports(self)
  File "/home/aled/anaconda2/envs/multiqc_exp/lib/python2.7/site-packages/multiqc-1.0.dev0-py2.7.egg/multiqc/modules/picard/AlignmentSummaryMetrics.py", line 23, in parse_reports
    for f in self.find_log_files(config.sp['picard']['alignment_metrics'], filehandles=True):
KeyError: 'alignment_metrics'

```
I don't know which picard stats file to upload o if one is useful please let me know!

**Comment by ewels** [comment]
Hi @aledj2,

Sounds like a conda installation error. Has anything ever worked with any input file from the conda installation? My guess is that it hasn't got a copy of the required [`search_patterns.yaml`](https://github.com/ewels/MultiQC/blob/master/multiqc/utils/search_patterns.yaml) file, which is required. When you install it manually from source, the file is present as it should be and everything works.

These installation difficulties are really difficult to track down as they're often hard for me to replicate. Sometimes they're just random stuff breaking too. Have you tried reinstalling with conda into a fresh environment? If it keeps happening, please let me know what system you're using, what command you used to create the environment and install MultiQC, and send over any logs from that process and I'll try to replicate the problem.

Cheers,

Phil

**Comment by aledj2** [comment]
Hi Phil, 
Thanks for the quick reply. I usually use a build of multiqc which I have built from source and am happy to continue to use this approach. I wanted to test a newer version to try setting the HsMetrics "Target Bases 30X" setting within the yaml file which I couldn't get to work on version 0.8. I thought downloading and installing a later version without using conda environments may interfere with my current install.

I'm not sure if I have got it to work using the conda installation, I feel like it worked once or twice. 

I've tried updating the conda install  to 0.9.1a0-py27_3 bioconda with the same result.

If I get stuck I'll open a new issue with a bit more evidence!

Thanks.
